interactions/api/models/guild.py

```diff
@@ -1600,6 +1600,21 @@ async def search_members(self, query: str, limit: Optional[int] = 1) -> List[Mem
         )
         return [Member(**member, _client=self._client) for member in res]
 
+    @property
+    def icon_url(self) -> str:
+        """
+        Returns the URL of the guild's icon.
+
+        :return: URL of the guild's icon (None will be returned if none of icon is set)
+        :rtype: str
+        """
+        url = "https://cdn.discordapp.com/"
+        if self.icon is not None:
+            url += f"icons/{int(self.id)}/{self.icon}.png"
```

Comment:
This doesn't account for animated guild icons.

Reply:
Ohhh, right, I forgot about this.